test/e2e/template/deployment-rbac.yaml

```diff
@@ -140,6 +140,9 @@ rules:
 - apiGroups: [""]
   resources: ["events"]
   verbs: ["create", "patch"]
+- apiGroups: [""]
+  resources: ["endpoints"]
+  verbs: ["create", "get", "list", "watch", "update", "patch"]
```

Comment:
Why do we need `create`, `update` and `patch`?

Reply:
hmm we need `create` and `get` for sure, the rest i am not so sure about.